src/sourmash/sbt_storage.py

```diff
@@ -19,6 +20,9 @@ def save(self, path, content):
     def load(self, path):
         pass
 
+    def save_exact(self, path, content):
```

Comment:
what do you think about changing the signature of `save` to
```
def save(self, path, content, *, overwrite=False):
```
and avoid `save_exact`?

Reply:
sure! I was thinking about that, too, and went with this for no particular reason - will adjust.